### ISSUE TITLE
Code Issue

### ISSUE THREAD

**Issue by phc4valid** [heyunh2015/diseaseBERT#3]
Hi , @heyunh2015 ,
the labels:batch[3] in the disease_knowledge_infusion_training.py was comment out in the original code.
The code would not run with the 'masked_lm_labels':  batch[3].
By replacing it to 'labels':      batch[3] ,it started training, but lead to worse results.
Is there anything to adjust or do I have any thing missing?
Really looking forward to your kind response. Thanks .Stay safe
`

inputs = {'input_ids':      batch[0],                      
#'attention_mask': batch[1],                     
 #'token_type_ids': batch[2] if args['model_type'] in ['bert', 'xlnet', 'albert'] else None,  # XLM don't use segment_ids                      'masked_lm_labels':  batch[3]}#'labels':         batch[3]}

`

**Comment by heyunh2015** [comment]
Hi,
We should use 'masked_lm_labels': batch[3], which is the right parameter name in Huggingface transformers 2.5.1...

If only using 'labels': batch[3] can work... I would guess you are using the latest version of transformers...

So... you could try to use Huggingface transformers 2.5.1.
